packages/twenty-server/src/engine/core-modules/billing/dtos/billing-product.dto.ts

```diff
@@ -1,12 +1,18 @@
-/* @license Enterprise */
+import { Field, InterfaceType, ObjectType } from '@nestjs/graphql';
 
-import { Field, ObjectType } from '@nestjs/graphql';
-
-import { type BillingPriceLicensedDTO } from 'src/engine/core-modules/billing/dtos/billing-price-licensed.dto';
-import { type BillingPriceMeteredDTO } from 'src/engine/core-modules/billing/dtos/billing-price-metered.dto';
-import { BillingPriceUnionDTO } from 'src/engine/core-modules/billing/dtos/billing-price-union.dto';
 import { BillingProductMetadata } from 'src/engine/core-modules/billing/types/billing-product-metadata.type';
+import { BillingPriceLicensedDTO } from 'src/engine/core-modules/billing/dtos/billing-price-licensed.dto';
+import { BillingPriceMeteredDTO } from 'src/engine/core-modules/billing/dtos/billing-price-metered.dto';
+import { BillingProductKey } from 'src/engine/core-modules/billing/enums/billing-product-key.enum';
 
+@InterfaceType({
```

Comment:
that's something new, what's the benefit over UnionType?

Reply:
`UnionType` is useful for describing two different entities with **nothing in common**.

`InterfaceType` is useful to describe two different entities **with common properties**. Here, the only specific field is `prices`